### ISSUE TITLE
Disable dynamic template char warning

### ISSUE THREAD

**Issue by alberto-fp-24** [sendgrid/sendgrid-nodejs#931]
Currently, when passing dynamicTemplateData that contains a quote or an ampersand, I am warned that I may need to use three brackets on the template.

```
Object.values(dynamicTemplateData).forEach(value => {
      if (/['"&]/.test(value)) {
        console.warn(DYNAMIC_TEMPLATE_CHAR_WARNING);
      }
    });
```
While it was originally very helpful, now it is cluttering my logs. Is there any way to disable this warning (other than overwriting `console.warn`)?

**Comment by thinkingserious** [comment]
Hello @alberto-fp-24,

Good call, I've added this to our backlog and also tagged it appropriately in case someone in our Node.js community would like to make a PR.

With Best Regards,

Elmer

**Comment by albertor24** [comment]
@thinkingserious thank you very much. 

I created a PR for this https://github.com/sendgrid/sendgrid-nodejs/pull/932

**Comment by vieira** [comment]
Hello, we are also having this problem since updating sendgrid/helpers from 6.3.0 to 6.4.0. We are using {{{ }}} but the warning is still logged. Is this the intended behaviour?

**Comment by thinkingserious** [comment]
Hello @vieira,

For now this is intended behavior; however, @albertor24 PR #932 gives you the option to hide that warning. It will be included in the next release of this SDK.

Thanks!

With Best Regards,

Elmer

**Comment by ErwinAI** [comment]
#965 is a duplicate of this issue.
@thinkingserious It seems that this PR hasn't made it into a release yet. It has been over three months now though. Is there any news on this?

**Comment by childish-sambino** [comment]
Fixed by #932 which was included in the 6.5.0 release.

**Comment by srinivas1186** [comment]
In sgMail.send function data parameter add hideWarnings : true value to suppress the warnings
